PointOfDispensingApp/src/store/store.js

```diff
@@ -0,0 +1,173 @@
+import Vuex from 'vuex'
+import Vue from 'vue'
+
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+    state: {
+        workflowState: 'INITIAL',
+        patientResource: {
+            patientId: '',
+            patientLastName: '',
+            patientFirstName: '',
+            patientDateOfBirth: '',
+            patientGender: '',
+            patientStreetAddress: '',
+            patientPreferredLanguage: ''
+        },
+        locationResource: {
+            locationId: '1234567890',
+            locationName: 'Western Lakes FD'
+        },
+        encounterResource: {
+            encounterStatus: '',
+            encounterTimeStamp: ''
+        },
+        immunizationResource: {
+            lotNumber: '',
+            expirationDate: '',
+            manufacturer: '',
+            doseQuantity: '',
+            doseNumber: '',
+            immunizationStatus: '',
+            immunizationTimeStamp: '',
+            healthcarePractitioner: 'White, Betty',
+            site: '',
+            route: 'Injection',
+            notes: '',
+            notAdministeredReason: ''
+        },
+        screeningResponses: {
+            vaccinationDecision: '',
+            patientInfoConfirmed: '',
+            consentFormSigned: '',
+            screeningCompleted: '',
+            factSheetProvided: '',
+            screeningComplete: false
+        }
+    },
+
+    getters: {
+        workflowState: state => {
+            return state.workflowState
+        },
+        isCheckInPageDisabled: state => {
```

Comment:
This series of logic is being used to determine which left menu options are enabled/disabled based on the patient's current workflow state (rather than what page the user is on).  The "is____PageDisabled" functions in conjunction with the left menu logic (see App.vue lines 34, 43, 52, 61, 70, 79) implement the state/logic table that I had copied/pasted into the PR (the red/green color-coded table).

At some point we may find a need to track which page the user is currently on, but I haven't encountered that need, yet (at least, not for this card).

Does that make sense?  Or am I misunderstanding your idea?

Reply:
I like the way the logic is written here.
I think it would be easier to read if there was a comment above the two cases where you use != that mentions that we are disabling the page except for when in the mentioned states (initial, discharged)

Reply:
> I like the way the logic is written here.
> I think it would be easier to read if there was a comment above the two cases where you use != that mentions that we are disabling the page except for when in the mentioned states (initial, discharged)

Great suggestion!  I added a comment to each of the lines of logic in this section.